grapher/stackedCharts/StackedAreaChart.tsx

```diff
@@ -486,10 +482,6 @@ export class StackedAreaChart
     }
 
     @computed get series() {
-        if (this.props.disableLinearInterpolation === true)
```

Comment:
Do we absolutely need `disableLinearInterpolation`?
I didn't re-implement it since interpolation is now applied in the constructor already...

Reply:
I would say absolutely. I think it's very important for reproducibility/"show your work", and it also makes testing interpolation super easy:

![Screen Shot 2020-11-05 at 9 32 01 AM](https://user-images.githubusercontent.com/74692/98287666-c4d06f00-1f49-11eb-8154-755309ef6c33.png)
![Screen Shot 2020-11-05 at 9 31 58 AM](https://user-images.githubusercontent.com/74692/98287719-d9146c00-1f49-11eb-93d3-4287dc169392.png)

Reply:
That being said, I don't think it matters where we turn it on or off, as long as it's easy to do at the story level.

Reply:
Nevermind, I added it back and it was easy. The story there might actually be useful.